core/model/modx/rest/modrest.class.php

```diff
@@ -473,7 +473,7 @@ public function toXml($parameters,$rootNode) {
      * @param array $parameters
      */
     protected function _populateXmlDoc(&$doc, &$node, &$parameters) {
-        /** @var $val DOMNode */
+        /** @var DOMnode/array $val */
```

Comment:
Alternatives should be separated using `|` instead of `/`. So correct way `DOMnode|array`